nightfall-optimist/src/event-handlers/rollback.mjs

```diff
@@ -132,14 +130,9 @@ async function rollbackEventHandler(data) {
   ]);
 
   await dequeueEvent(2); // Remove an event from the stopQueue.
-  // A Rollback triggers a NewCurrentProposer event which shoudl trigger queue[0].end()
+  // A Rollback triggers a NewCurrentProposer event which should trigger queue[0].end()
```

Comment:
These comments aren't necessary anymore.

Reply:
fffd3a10 Clearly I didn't understand which comments were linked to what code... :sweat_smile: